css/bootstrap.scss

```diff
@@ -87,6 +87,15 @@ div.main-content {
   background: white;
   overflow: visible;
 
+  .version-warning {
+    font-weight: bold;
+    letter-spacing: 2px;
+    border: 8px solid theme-color("warning");
+    background-color: theme-color("warning");
+    margin: auto;
+    text-transform: uppercase;
```

Comment:
If there isn't a case where we will want this uppercase, I would suggest that we just uppercase the actual content and remove this.

Reply:
I have now removed `text-transform: uppercase;` and the v in the version is lowercase, but it is simpler, and looks fine.